src/actions/users.js

```diff
@@ -224,6 +224,8 @@ export function getProfiles(offset, limit = General.PROFILE_CHUNK_SIZE) {
             return null;
         }
 
+        const profiles = profileListToMap(profilesList);
```

Comment:
Oh sorry, I still want it a map in the store, but I meant we should could pass it as an array in the action since I'm almost certain most of those reducers just loop over the map anyway